settings.gradle.kts

```diff
@@ -7,11 +7,22 @@ include("save-plugins:fix-plugin")
 include("save-plugins:warn-plugin")
 include("save-reporters")
 include("save-common-test")
-includeBuild("sarif4k")
 
 dependencyResolutionManagement {
     repositories {
         mavenCentral()
+        maven {
+            url = uri("https://maven.pkg.github.com/analysis-dev/sarif4k")
+            val gprUser: String? by settings
```

Comment:
please notify everyone in our team about this change, that they need to set tokens, ok?